packages/shared-data/plans.ts

```diff
@@ -130,20 +115,21 @@ export const plans: PricingInformation[] = [
     href: 'https://forms.supabase.com/enterprise',
     description: 'For large-scale applications managing serious workloads.',
     features: [
-      `Designated Support manager & SLAs`,
-      `On-premise support`,
-      `24×7×365 premium enterprise support`,
-      'Custom Security Questionnaires',
-      `Private Slack channel`,
+      'Designated Support manager',
```

Comment:
features and featuresPartner should be the same here, so featuresPartner would need to be aligned